sdk/storage/storage-blob/package.json

```diff
@@ -135,6 +135,7 @@
     "@azure/test-utils-recorder": "^1.0.0",
     "@azure/test-utils-perfstress": "^1.0.0",
     "@microsoft/api-extractor": "7.7.11",
+    "@opentelemetry/api": "^1.0.1",
```

Comment:
This dependency is not included in main branch. Do we need to introduce this change to main branch?

Reply:
In the main branch, this is probably not needed - because we have the @azure/test-utils package which houses this logic. Unfortunately this release was created before @azure/test-utils was added so I had to inline these test helpers